tests/test_app_create_edit_delete.py

```diff
@@ -50,5 +69,4 @@ def test_app_create_edit_delete(client):
 
     # delete the test device webp dir
     db.delete_device_dirs(device_id)
-    device_dir = Path("tronbyt_server") / "webp" / device_id
-    assert not device_dir.is_dir()
+    assert not os.path.isdir(f"tronbyt_server/webp/{device_id}")
```

Comment:
Could we stick to pathlib as the idiomatic library for such operations with modern Python?

Reply:
yeah sure. those were left over from before.